src/cosmosdb-preview/azext_cosmosdb_preview/tests/latest/test_cosmosdb-pitr_scenario.py

```diff
@@ -820,3 +820,64 @@ def test_cosmosdb_system_identity_restore(self, resource_group):
 
         restored_account_defaultIdentity = restored_account['defaultIdentity']
         assert user_id_2 in restored_account_defaultIdentity
+
+        public_network_access = restored_account['publicNetworkAccess']
+        assert public_network_access == 'Disabled'
+
+    @AllowLargeResponse()
+    @ResourceGroupPreparer(name_prefix='cli_test_cosmosdb_cross_region_restore', location='eastus2')
+    def test_cosmosdb_xrr(self, resource_group):
+        col = self.create_random_name(prefix='cli', length=15)
+        db_name = self.create_random_name(prefix='cli', length=15)
+        source_acc = self.create_random_name(prefix='cli-continuous30-', length=25)
+        target_acc = source_acc + "-restored"
+        loc = 'eastus2'
+        target_loc = 'centralus'
+
+        self.kwargs.update({
+            'acc': source_acc,
+            'db_name': db_name,
+            'restored_acc': target_acc,
+            'col': col,
+            'loc': loc,
+            'target_loc': target_loc
+        })
+
+        self.cmd('az cosmosdb create -n {acc} -g {rg} --backup-policy-type Continuous --continuous-tier Continuous7Days --locations regionName={loc} failoverPriority=0 isZoneRedundant=False --locations regionName={target_loc} failoverPriority=1 isZoneRedundant=False --kind GlobalDocumentDB')
+        account = self.cmd('az cosmosdb show -n {acc} -g {rg}').get_output_in_json()
+        print(account)
+
+        assert account['location'] == 'East US 2'
+
+        self.kwargs.update({
+            'ins_id': account['instanceId']
+        })
+
+        # Create database
+        self.cmd('az cosmosdb sql database create -g {rg} -a {acc} -n {db_name}')
+
+        # Create container
+        self.cmd('az cosmosdb sql container create -g {rg} -a {acc} -d {db_name} -n {col} -p /pk ').get_output_in_json()
+
+        restorable_database_account = self.cmd('az cosmosdb restorable-database-account show --location {loc} --instance-id {ins_id}').get_output_in_json()
+        print(restorable_database_account)
+
+        account_creation_time = restorable_database_account['creationTime']
+        creation_timestamp_datetime = parser.parse(account_creation_time)
+        restore_ts = creation_timestamp_datetime + timedelta(minutes=61)
+        import time
+        time.sleep(3662)

```

Comment:
put comments on why we are sleeping this arbitrary number of seconds/minutes

Reply:
added